src/sdk/base/VerifyingPaymasterAPI.ts

```diff
@@ -28,24 +28,24 @@ export class VerifyingPaymasterAPI extends PaymasterAPI {
     this.context = context;
   }
 
-  async getPaymasterAndData(userOp: Partial<UserOperationStruct>): Promise<PaymasterResponse> {
+  async getPaymasterAndData(userOp: Partial<UserOperation>): Promise<PaymasterResponse> {
     // Hack: userOp includes empty paymasterAndData which calcPreVerificationGas requires.
     try {
       // userOp.preVerificationGas contains a promise that will resolve to an error.
       await ethers.utils.resolveProperties(userOp);
       // eslint-disable-next-line no-empty
     } catch (_) { }
-    const pmOp: Partial<UserOperationStruct> = {
+    const pmOp: Partial<UserOperation> = {
       sender: userOp.sender,
       nonce: userOp.nonce,
-      initCode: userOp.initCode,
+      factoryData: userOp.factoryData,
       callData: userOp.callData,
       callGasLimit: userOp.callGasLimit,
       verificationGasLimit: userOp.verificationGasLimit,
       maxFeePerGas: userOp.maxFeePerGas,
       maxPriorityFeePerGas: userOp.maxPriorityFeePerGas,
       // A dummy value here is required in order to calculate a correct preVerificationGas value.
-      paymasterAndData: DUMMY_PAYMASTER_AND_DATA,
+      paymasterData: DUMMY_PAYMASTER_AND_DATA,
```

Comment:
`paymasterData` can not be used without `paymaster`, `paymasterVerificationGasLimit`, `paymasterPostOpGasLimit` fields. pls add them too